tests/unit/test_vcs.py

```diff
@@ -98,14 +98,16 @@ def test_rev_options_make_new():
     assert new_options.vc_class is Git
 
 
-def test_looks_like_hash():
-    assert looks_like_hash(40 * 'a')
-    assert looks_like_hash(40 * 'A')
-    # Test a string containing all valid characters.
-    assert looks_like_hash(18 * 'a' + '0123456789abcdefABCDEF')
-    assert not looks_like_hash(40 * 'g')
-    assert not looks_like_hash(39 * 'a')
-    assert not looks_like_hash(41 * 'a')
+@pytest.mark.parametrize('sha, expected', [
+    ((40 * 'a'), True),
+    ((40 * 'A'), True),
+    ((18 * 'a' + '0123456789abcdefABCDEF'), True),
```

Comment:
It'd be nice to preserve all comments. :)

Reply:
```suggestion
     # Test a string containing all valid characters.
    ((18 * 'a' + '0123456789abcdefABCDEF'), True),
```

Reply:
fair enough

Reply:
Looks like there was an extra space before the # on line 105. I am not sure if it was the right way to do it but i did another push with the space omitted.

Reply:
Whoops! My bad.